### DISCUSSION TITLE
what is default username and password

### DISCUSSION THREAD

**Discussion by luridarmawan** [Quansight/ragna#167] [Q&A] [answered]
I have install ragna with default config option and success. What is default username and password? How to setting up?

![Screenshot 2023-11-07 064654](https://github.com/Quansight/ragna/assets/3904989/4c79069a-6c95-4ee1-bdb0-6b1c2396167c)

**Comment by pmeier** [comment, marked as answer]
Hey @luridarmawan and thanks for the question. Indeed it is no obvious and something that we want to change in the near future.

We are using [`RagnaDemoAuthentication`](https://ragna.chat/en/stable/references/python-api/#ragna.core.RagnaDemoAuthentication) here. Meaning any matching username / password will work. For example, you could leave both fields blank or use `"admin"` / `"admin"`.

**Reply by pmeier** [reply]
I have opened #168 to have this inside the documentation.

**Reply by pmeier** [reply]
This is now documented under https://ragna.chat/en/stable/generated/tutorials/gallery_web_ui/.
